### PR TITLE
Version exposed in the API

### PR DESCRIPTION
**Pull request by hilde2**
# Pull Request description

### How to verify it
Just GET http://localhost:8800/server/info with a rest client like insomnia.

### CODE REVIEW

File: config/info.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+// YorcVersion is the used version of YORC
+var YorcVersion = "Must Be defined by MakeFile"
```
Comment (loicalbertin) [inline config/info.go]: Hum not sure that the `config` package is the right place for this. `config` is for things that are configurable...

File: commands/version.go [hunk 2]
```diff
@@ -18,16 +18,15 @@ import (
...
 
 	"github.com/spf13/cobra"
-
+	"github.com/ystia/yorc/v4/config"
 	"github.com/ystia/yorc/v4/helper/consulutil"
 )
 
-var version = "unknown phantom version"
-
 var gitCommit = "dev version"
```
Comment (loicalbertin) [inline commands/version.go]: I would keep the `version` and `gitCommit` in the same place they describe the same thing.

File: rest/http.go [hunk 3]
```diff
@@ -143,6 +143,7 @@ func NewServer(configuration config.Configuration, client *api.Client, shutdownC
 
 func (s *Server) registerHandlers() {
 	commonHandlers := alice.New(telemetryHandler, loggingHandler, recoverHandler)
+	s.router.Get("/info/yorcVersion", commonHandlers.Append(acceptHandler("application/json")).ThenFunc(s.getInfoHandler))
```
[the remark this answers is not in the document]
Comment (loicalbertin) [reply rest/http.go]: I'm not sure that `/info` is the best endpoint path.
Maybe we should have `/server/info` this way we will be able to add some other endpoints to control the server under the  ̀/server` path.
@ystia/yorc-dev WDYT?

Also we may move `/health` under `/server/health` as it is giving info on the local server only. @ystia/yorc-dev WDYT?

**Comment by loicalbertin** [comment]
Also please have a look on Sonar issues :smile:

File: commands/version.go (hunk 2 above)
Comment (loicalbertin) [reply commands/version.go]: `gitCommit`should somehow be part of the version information  retrieved by the API

File: config/info.go (hunk 1 above)
Comment (hilde2) [reply config/info.go]: I didn't find any better fitting package for this. `config` looked liked the the least worst. 
So maybe a new one must be created ? Maybe `info` or `others`

Comment (loicalbertin) [reply config/info.go]: And what about a `server/info` package for info about a server? 
@ystia/yorc-dev WDYT?

File: rest/http.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (stebenoist) [reply rest/http.go]: Good idea !

File: config/info.go (hunk 1 above)
Comment (stebenoist) [reply config/info.go]: Yep

**Pull request merged**